docs/tutorial/part-three/index.md

```diff
@@ -19,7 +19,7 @@ Let's dive in and explore Gatsby layouts.
 
 ## Install a starter
 
-First, create a new site for this part of the tutorial (and, as we mentioned in Part Two, at this point it's probably a good idea to close the terminal window(s) and project files from previous parts of the tutorial, to keep things clean on your desktop). We'll use a "hello world" starter again:
+As we mentioned in Part Two, at this point it's probably a good idea to close the terminal window(s) and project files from previous parts of the tutorial, to keep things clean on your desktop. Then, open a new terminal window and and run the following commands to create a new Gatsby site in a directory called tutorial-part-three and then move to this new directory:
```

Comment:
It appears that in most of the tutorial files the wrapping is done manually, the paragraphs don’t auto-wrap (see lines 10-16 in this file). Should we worry about the wrapping length or is this done automagically by Gatsby?